openlibrary/macros/FulltextSuggestionSnippet.html

```diff
@@ -19,8 +19,8 @@
           </a>&nbsp;
           <div class="fsi-snippet__quotation-mark">$_('❞')</div>
           $if page:
-          <div class="fsi-snippet__page-num" href="$ia_base_url/details/$ia">
+          <span class="fsi-snippet__page-num" href="$ia_base_url/details/$ia">
```

Comment:
this may still need to be indented
```suggestion
            <span class="fsi-snippet__page-num" href="$ia_base_url/details/$ia">
```